Formula/s/spatialite-tools.rb

```diff
@@ -23,9 +23,23 @@ class SpatialiteTools < Formula
   end
 
   depends_on "pkg-config" => :build
+
   depends_on "libspatialite"
   depends_on "proj"
+  depends_on "readline"
   depends_on "readosm"
+  depends_on "sqlite"
+
+  uses_from_macos "expat"
+  uses_from_macos "libxml2"
```

Comment:
Make this `depends_on` and you can remove the one inside the `on_macos` block

Reply:
👍